v3/mocks/TestOperations_DeleteIdempotenceIdentifers.yaml

```diff
@@ -0,0 +1,45 @@
+version: api.keploy.io/v1beta2
+kind: Http
+name: TestOperations_DeleteIdempotenceIdentifers
+spec:
+    metadata:
+        name: Http
+        operation: DELETE
+        type: HTTP_CLIENT
+    req:
+        method: DELETE
+        proto_major: 1
+        proto_minor: 1
+        url: https://pc.dev.nkp.sh:9440/api/nutanix/v3/idempotence_identifiers/testclient
```

Comment:
so, the FQDN this records actually have to be the env you used to record it. Can we replace it with `prism-test.nutanix.com` ?

Reply:
Done in https://github.com/nutanix-cloud-native/prism-go-client/pull/204/commits/4e2cb80834d25165c784bad8a01242892fc1f06f